docs/ace_api.md

```diff
@@ -0,0 +1,337 @@
+
+## ACE API Endpoints
+
+ACE includes API endpoints for some of its most frequently used functions.
+
+## API Reference
```

Comment:
Although, at a high level, the functions are the same, there are subtle differences between the Python and Golang versions of ACE. I think we shouldn't be adding the Python ACE-related docs to this repo.